### PR TITLE
add polish commands

### PR DESCRIPTION
**Pull request by kompowiec2**

### CODE REVIEW

**Comment by lkdjiin** [comment]
Maybe we are the only two people in the world using both logo into vim ;)

**Comment by lkdjiin** [comment]
Please update version to 0.5.0 before I merge…

File: logo.vim [hunk 1]
```diff
@@ -18,6 +18,10 @@ syn keyword logoKeyword bye end stop output
 syn keyword logoConditional if ifelse iffalse iff iftrue ift test and or not
 syn keyword logoRepeat repeat for forever foreach
 
+" Polish keywords (imagine)
+syn keyword LogoKeyword naprzód wstecz prawo lewo powtórz np ws pw lw hop to już cs pokaż czekaj
+syn keyword LogoConditional ustalKolPis ustalKolMal ustalGrubość ukp ukm ugp jdl [pod]nieś opu[ść] zam[aluj] okrąg koło kwa[drat] trójkąt słuchaj powiedz lista
+syn keyword LogoRepeat for nowaklasa pisz[wiersz] nowy przypisz nazwij jdl
```
Comment (lkdjiin) [inline logo.vim:24]: `for` is already in the list so you can safely remove it, isn't it?

**Comment by kompowiec** [comment]
Yeah, of course, this was m.e. mistake. Sorry, but i forgot pass (typical) and forget about project.

"Maybe we are the only two people in the world using both logo into vim ;)" - 
I've never seen logo on the eyes, unless in Polish State school, where there is such a pathology.

**Pull request merged**